apps/frontend/src/pages/DashboardPage.css

```
@@ -0,0 +1,404 @@
+/* DashboardPage.css */
+
+/* Add your styles here */
+
+@import url('https://fonts.googleapis.com/css2?family=Zen+Maru+Gothic:wght@500;700;800&display=swap');
```

Stylelint (17.4.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Zen+Maru+Gothic:wght@500;700;800&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Zen+Maru+Gothic:wght@500;700;800&display=swap'" (import-notation)

(import-notation)